engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregator.java

```diff
@@ -235,7 +239,7 @@ private void aggregationBasedOnEvent(EventAggregationCriteria eventAggregationCr
                     AbstractEmailPayloadAggregator aggregator = aggregated
                         .computeIfAbsent(recipient, ignored -> EmailPayloadAggregatorFactory.by(eventAggregationCriteria, start, end));
                     // It's aggregation time!
-                    EmailAggregation eventDataToAggregate = new EmailAggregation(aggregation.getOrgId(), eventAggregationCriteria.getBundle(), eventAggregationCriteria.getApplication(), new JsonObject(aggregation.getPayload()));
+                    EmailAggregation eventDataToAggregate = new EmailAggregation(aggregation.getOrgId(), eventAggregationCriteria.getBundle(), eventAggregationCriteria.getApplication(), baseTransformer.toJsonObject(aggregation));
```

Comment:
We were transforming `aggregation.getPayload()` (which is actually the return value of `Event#getPayload`) into a `JsonObject` in the old code. Now, we're transforming `aggregation` (the `Event` object) into a `JsonObject`.

I would expect that change to break the implementation logic in several aggregators. Is it a safe change?